src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+* {
```

Comment:
Don't use * selector (it impacts performance), better use ``html`` or ``body`` tag.

![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/dee89a50-9e97-4c19-b57a-653f59ceab7e)

Reply:
@LitviVit I read your comment carefully and that's why I added a screenshot of the checklist: using the generic selector "*" is not a good practice, so it's best to avoid using this approach.
Each mentor has personal experience depending on the company where he works, so I focus on the general checklist during the inspection so that there are fewer questions.

Reply:
@roman-mirzoian ok. Understood. So what do you think about this solution?
![image](https://github.com/mate-academy/layout_product-cards/assets/168312293/043d913c-4063-476e-9358-daedde809a03)
I found it while researching this question on stackoverflow

Reply:
@LitviVit This solution is better than specifying this rule directly 👍